docs/import/manage_disaster.js

```diff
@@ -404,22 +417,32 @@ function initializeScoreSelectors(states) {
   // Initialize headers.
   removeAllButFirstFromRow(headerRow);
   for (const state of states) {
-    headerRow.append(createTd().html(state + ' Assets'));
+    headerRow.append(createTd().text(state + ' Assets'));
   }
 
   // For each asset type, add select for all assets for each state.
   for (const scoreAssetType of scoreAssetTypes) {
-    const id = assetSelectionRowPrefix + scoreAssetType[0];
+    const type = scoreAssetType[0];
+    const id = assetSelectionRowPrefix + type;
     const propertyPath = scoreAssetType[1];
+    const expectedColumns = scoreAssetType[3];
     const row = $('#' + id);
     removeAllButFirstFromRow(row);
     for (const state of states) {
       if (stateAssets.get(state)) {
         const statePropertyPath = propertyPath.concat([state]);
-        row.append(createTd().append(addAssetDataChangeHandler(
+        const select =
             createAssetDropdown(stateAssets.get(state), statePropertyPath)
-                .prop('id', 'select-' + id + '-' + state),
-            statePropertyPath)));
+                .prop('id', 'select-' + id + '-' + state)
+                .on('change',
+                    (event) => onNonDamageAssetSelect(
+                        event, statePropertyPath, expectedColumns, type, state))
+                .prop('style', 'border:2px');
+        row.append(
+            createTd().append($(document.createElement('span'))
```

Comment:
Why can't you add hover text to the select directly? When I give it a title attribute, it seems to hover fine.
![hover-screenshot](https://user-images.githubusercontent.com/10134896/70753655-42bcd380-1d03-11ea-9312-801e84f11d1f.png)